docs/pages/blog/date-pickers-stable-v5.md

```diff
@@ -39,14 +39,14 @@ For example, the `MonthPicker`’s prop `onMonthChange` and the `YearPicker`’s
 
 We also added new customizable slots:
 
-- The `ActionBar` allows you to customize the actions below the picker. It replaces the props `clearable`, `showTodayButton`, `cancelText`, `okText`
-- The `PaperContent` allows you to add custom elements in the Desktop view
+- The `ActionBar` enables customizing the actions below the picker. It replaces the props `clearable`, `showTodayButton`, `cancelText`, `okText`
```

Comment:
I think "lets you" is a preferable substitute for "allows you to" in this context.

```suggestion
- `ActionBar` lets you customize the actions below the picker. It replaces the props `clearable`, `showTodayButton`, `cancelText`, `okText`
```